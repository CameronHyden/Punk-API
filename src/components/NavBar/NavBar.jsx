import React from "react";
import CheckboxFilters from "../CheckboxFilters/CheckboxFilters";
import "./NavBar.scss";
import SearchBar from "../SearchBar/SearchBar";
import brewDog from "../../Assets/Images/brewDog.png";

const NavBar = ({
  handleAbvCheckbox,
  handleSearchInput,
  handleDate,
  handlePh,
}) => {
  return (
    <div className="navbar">
      <img className="brewDog-logo" src={brewDog} alt="" />
      <SearchBar handleSearchInput={handleSearchInput} />
      <CheckboxFilters
        handleAbvCheckbox={handleAbvCheckbox}
        handleDate={handleDate}
        handlePh={handlePh}
      />
    </div>
  );
};

export default NavBar;
