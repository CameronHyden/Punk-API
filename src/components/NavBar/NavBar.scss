.filters__container {
  box-shadow: rgba(0, 0, 0, 0.16) 0px 1px 4px;
  position: fixed;
  background-color: #ffffff;
  width: 100%;
  overflow: hidden;
  z-index: 2;
  height: 130px;
  display: flex;
  justify-content: space-between;
  align-items: center;
}
.brewDog-logo {
  height: 150px;
}
