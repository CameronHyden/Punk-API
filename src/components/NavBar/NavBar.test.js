import { render, screen } from "@testing-library/react";
import userEvent from "@testing-library/user-event";
import NavBar from "./NavBar";

it("should match NavBar component snapshot", () => {
  const { container } = render(<NavBar />);

  expect(container).toMatchSnapshot();
});
