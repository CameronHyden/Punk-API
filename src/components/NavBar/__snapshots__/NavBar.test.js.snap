// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match NavBar component snapshot 1`] = `
<div>
  <div>
    <label
      for="searchBar"
    >
      search for your favorite beer
    </label>
    <input
      class="searchBar"
      type="text"
      value=""
    />
  </div>
</div>
`;
